.github/workflows/build-and-push-image-test.yml

```diff
@@ -0,0 +1,73 @@
+name: Continuous delivery (test)
+
+on:
+  push:
+    branches:
```

Comment:
This would get deployed automatically to test when we merge to main. Is that what we want to happen?

Reply:
Even though the workflow runs on every push, [Line 17](https://github.com/DFE-Digital/trams-data-api/pull/297/files/04394b4702e24fe68fbabd08ccf425673d33f020#diff-743cd31f975dd7d580777082c4be6e8364a36f3fb98097561bb0c03ca05f0f0cR17) ensures it will only ever checkout the GitHub tag `prerelease`.